### DISCUSSION TITLE
Backup and mtd blocks

### DISCUSSION THREAD

**Discussion by Mrivilla** [roleoroleo/yi-hack-Allwinner-v2#212] [Q&A]
Hi all I had a problem with the wifi and fortunately the system did a mtdblock6 backup in the SD, so I just restored.
Then I realized I never did full dump, but when I did... surprise!!! I do not have the same mtd structure shown in the Wiki. 
This is the mine one:
![image](https://user-images.githubusercontent.com/13434459/123813294-0fe22c80-d8f5-11eb-8a1a-a935856a0343.png)

Is there something wrong? indeed my Hack works without SD card connected. My camera is a 9FUS.
Thanks in advance.
Marcos.

**Comment by roleoroleo** [comment]
Your cam is allwinner not allwinner-v2.
